.nvmrc

```diff
@@ -0,0 +1 @@
+v21.6
```

Comment:
I would use the LTS version instead.

Reply:
Me too, I added this to align with what's in package.json. Will update